bin/python-noenv.sh

```diff
@@ -25,4 +25,5 @@
 # along with vsc-install. If not, see <http://www.gnu.org/licenses/>.
 #
 ### END OF HEADER
-env -i /usr/bin/python "$@"
+# the -s makes sure the user site package is also ignored
```

Comment:
Add same comment as in in python-stripped-env, and add a comment to explain why `env -i` is used?